metrotranscom/doorway
# fix: availability type filter

This PR addresses #720


## Description

Adds availability filter for landing page and listings page

## How Can This Be Tested/Reviewed?

to have `openWaitlist` listing we need to select on partners side `What is the listing availability?` -> open waitlist. And then on application tab `Do you want to show a waitlist size?` -> Yes

Then if you have both listings with `isWaitlistOpen` and `available types` test it filtering works, both for landing, and listings page.
Make sure to test with other filters

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
